tests/hypershift/setup-vpc.tf

```diff
@@ -0,0 +1,66 @@
+variable "aws_region" {
+  type        = string
+  description = "The region to create the ROSA cluster in"
+  default     = "us-east-2"
+
+  validation {
+    condition     = contains(["us-east-2", "us-west-2"], var.aws_region)
```

Comment:
If we will need to have other regions ? In stage/prod the check need to support it also

Reply:
moved to python